packages/components/Footer.tsx

```diff
@@ -1,64 +1,51 @@
 "use client";
 
-import { Stack, Text, Image, Grid } from "@chakra-ui/react";
-import { css } from "@emotion/react";
-
-import { useI18n } from "@packages/features/i18n-context";
-import { PIX_KEY, images } from "@packages/config/site";
 import { Logo } from "@packages/components/icons";
+import { PIX_KEY, images } from "@packages/config/site";
+import { useI18n } from "@packages/features/i18n-context";
 
 import Section from "./Section";
 import SocialIconLinks from "./SocialIconLinks";
 
 export default function Footer() {
   return (
     <Section className="bg-neutral-950 px-6 py-6 items-center">
-      <Grid gridTemplateColumns={{ base: "1fr", sm: "repeat(3, 1fr)" }} color="gray.50" css={centerGridCells}>
+      <div className="grid grid-cols-3 text-gray-50">
         <PodCodarLogo />
         <Copyrights />
         <Pix />
-      </Grid>
+      </div>
     </Section>
   );
 }
 
 function PodCodarLogo() {
   return (
-    <Stack>
+    <div className="flex flex-col items-center gap-2">
       <Logo size="large" />
-    </Stack>
+    </div>
   );
 }
 
 function Pix() {
   const { t } = useI18n("footer");
   return (
-    <Stack>
-      <Text>{t("contribution")}</Text>
-      <Text fontSize="sm" color="#718096">
-        {PIX_KEY}
-      </Text>
-      <Image src={images.pixQRCode} width="40%" alt={t("contribution")} />
-    </Stack>
+    <div className="flex flex-col items-center gap-2">
+      <p>{t("contribution")}</p>
+      <p className="text-sm text-zinc-500">{PIX_KEY}</p>
+      <img src={images.pixQRCode} width="40%" alt={t("contribution")} />
```

Comment:
I think we need to use `<Image>` - https://nextjs.org/docs/pages/api-reference/components/image

Reply:
Thanks for the redirection, changed in: https://github.com/podcodar/webapp/pull/189/commits/3511cd519bd0a51b603de256eab747335ec798ad